### DISCUSSION TITLE
Is it possible to use FlexLayout as simply a javascript library?

### DISCUSSION THREAD

**Discussion by abalter** [caplin/FlexLayout#358] [Q&A]
I'm not a react programmer and don't want to be. I also don't want to need a server-side process to run my app.

Is there a way to use this as a plain javascript library after compilation?

I was able to run `FlexLayout/demos/lv0.7demo` as an html/css/js app. But it runs from a file called `demo.js`  which has >43k lines, four times larger than jquery v3 uncompressed.

Is there something like a flexlayout.js library I can use?

**Comment by shuvro111** [comment]
@abalter It's not possible to use **FlexLayout** as an standalone JavaScript Libraray  at this moment because **FlexLayout**'s only dependencies are [React](https://reactjs.org) and [uuid](https://www.npmjs.com/package/uuid).
